### ISSUE TITLE
(2) $2k/month - Facebook Page Monetization

### ISSUE THREAD

**Issue by szwnba** [szwnba/affweb#194]
[![](https://afflift.com/f/data/avatars/m/58/58177.jpg?1641647008)](https://afflift.com/f/members/tamas.58177/) 

*   [Add bookmark](https://afflift.com/f/posts/115792/bookmark "Add bookmark")
*   [#1](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115792)

Some of you saw my post about this a few weeks ago so I've finally decided to document it this time as I've failed/abandoned it a few times in the past, mainly because it's slow and boring when you work alone. I'll now try again with the help of this FA /community and hopefully, get it to $2k monthly.

The page is over 8 years old, mainly built during the teespring gold rush. Then I didn't do much with it but kept seeing good engagement on it.

Now with the Reels launch to compete with tiktok, I've started testing it and noticed the reach is awesome on them, and the best part about it is that they are pushing it to non-followers a lot, just like tik tok. I already have a reel with 2.2M reach.

![](https://afflift.com/f/attachments/1680289121492-png.37172/)

I also like all the new tools they're launching to help creators with content strategies, analytics and monetization, like the Meta Business Suite.

Niche: dogs  
FB Page Followers: 400k  
FB Group members: 30k  
Instagram: 5k followers  
GEO: US/T1  
Goal: $2k/month

![](https://afflift.com/f/attachments/1680290188339-png.37173/)

![](https://afflift.com/f/attachments/1680287938520-png.37171/)

![](https://afflift.com/f/attachments/1680290242910-png.37174/)

![](https://afflift.com/f/attachments/1680290297065-png.37175/)

A competitor built a whole ecom brand around it, driving traffic from the page. Another one has a blog monetized with adsense and I think taboola or [revcontent](https://afflift.com/f/link-directory/revcontent.107/).

I'll probably start with a simple blog because I can easily get it to 2-3-5k visits/day (I know what kind of content they like) and then test a few monetization strategies. (Anth, Varun pm me for exclusive native placements😅)

Content will be mostly user-generated and AI. I'll have more details when I get to work lol.

That's it for now. Updates will come bi-weekly / monthly after the initial blog setup/content strategy etc.

Do any of you have social pages? What do you think of my plans?

 [![](https://afflift.com/f/data/avatars/m/0/571.jpg?1672675693)](https://afflift.com/f/members/dood.571/) 

*   [Add bookmark](https://afflift.com/f/posts/115793/bookmark "Add bookmark")
*   [#2](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115793)

Good luck!

Maybe you could try this to "convert" them to push subs?

![](https://lh6.googleusercontent.com/UpEZ4ezVIL2s-2OjSsKosut4tQDRrdvzfdviaCQ2WxynJE7NkcTlY9RLN9X0k6SzTknbpkBqvsv7M8QlVfFQ3sFrHsxiDC7obiKkC-XC2YZGDCep_uzBU6XTQi5IvbeM90VzR6SNL2lIYGVVZnQf7vlzHgMRLlq4N3JrbX9e75lSw9YBsmpEpTp8v9_lNw)

 [![](https://afflift.com/f/data/avatars/m/0/289.jpg?1617383031)](https://afflift.com/f/members/deadz.289/) 

*   [Add bookmark](https://afflift.com/f/posts/115799/bookmark "Add bookmark")
*   [#3](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115799)

Have you considered building a list? You just need a bridge/quiz optin page (angle is best/exclussive/massive discount deals, etc) -> [clickbank offers](https://www.youtube.com/watch?v=ua8whbw1QBU), [barkbox](https://www.barkbox.com/affiliate), squaredance/jumbleberry also has HQ [offers like this one](https://www.nomnomnow.com/) (\+ they have creatives for ads). And you can also optinto the clickbank offers yourself to see what other offers their lists are promoting, etc.

Also maybe [Shinez](https://shinez.io/) has viral content in the pet niche, so that's another possible revenue stream for you there.

 [![](https://afflift.com/f/data/avatars/m/58/58177.jpg?1641647008)](https://afflift.com/f/members/tamas.58177/) 

*   [Add bookmark](https://afflift.com/f/posts/115800/bookmark "Add bookmark")
*   [#4](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115800)

> Have you considered building a list? You just need a bridge/quiz optin page (angle is best/exclussive/massive discount deals, etc) -> [clickbank offers](https://www.youtube.com/watch?v=ua8whbw1QBU), [barkbox](https://www.barkbox.com/affiliate), squaredance/jumbleberry also has HQ [offers like this one](https://www.nomnomnow.com/) (\+ they have creatives for ads). And you can also optinto the clickbank offers yourself to see what other offers their lists are promoting, etc.
> 
> Also maybe [Shinez](https://shinez.io/) has viral content in the pet niche, so that's another possible revenue stream for you there.

Wow Deadz, thanks so much for all the new ideas!! I'll research them all.

 [![](https://afflift.com/f/data/avatars/m/77/77831.jpg?1681746680)](https://afflift.com/f/members/lazyguy.77831/) 

*   [Add bookmark](https://afflift.com/f/posts/115805/bookmark "Add bookmark")
*   [#6](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115805)

I've an Instagram page which is dead for 3 years now and has ~10k followers. Do you think Facebook Reels can revive it?

 [![](https://afflift.com/f/data/avatars/m/58/58177.jpg?1641647008)](https://afflift.com/f/members/tamas.58177/) 

*   [Add bookmark](https://afflift.com/f/posts/115806/bookmark "Add bookmark")
*   [#7](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115806)

> I've an Instagram page which is dead for 3 years now and has ~10k followers. Do you think Facebook Reels can revive it?

I would definitely try it. Find some tiktok virals in your niche and test a few.

 [![](https://afflift.com/f/data/avatars/m/34/34738.jpg?1667500929)](https://afflift.com/f/members/wealthyplayboi.34738/) 

*   [Add bookmark](https://afflift.com/f/posts/115815/bookmark "Add bookmark")
*   [#8](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115815)

The accounts are assets. Great that you kept them and kept at it. I'm excited to see how you monetize this and scale it.

> I've an Instagram page which is dead for 3 years now and has ~10k followers. Do you think Facebook Reels can revive it?

Yes

 [![](https://afflift.com/f/data/avatars/m/19/19855.jpg?1643690936)](https://afflift.com/f/members/akif.19855/) 

*   [Add bookmark](https://afflift.com/f/posts/115818/bookmark "Add bookmark")
*   [#9](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115818)

Hey [@Tamas](https://afflift.com/f/members/58177/)

Send them to my super optimized exclusive smartlink for the highest EPC. Contact me for this one time opportunity.

Only today. Last spot!

 [![](https://afflift.com/f/data/avatars/m/0/1.jpg?1672673274)](https://afflift.com/f/members/luke.1/) 

*   [Add bookmark](https://afflift.com/f/posts/115839/bookmark "Add bookmark")
*   [#10](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115839)

> (Anth, Varun pm me for exclusive native placements😅)

Varun looked at his [RevContent](https://afflift.com/f/link-directory/revcontent.107/) stats and despite my low volume as a publisher, he has bought clicks from me unintentionally lol

 [![](https://afflift.com/f/data/avatars/m/0/1.jpg?1672673274)](https://afflift.com/f/members/luke.1/) 

*   [Add bookmark](https://afflift.com/f/posts/115840/bookmark "Add bookmark")
*   [#11](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115840)

Very interested to see how this goes Tamas. I have never monetized video because all of my niches are pretty copyright prone (I have received many C&Ds over the years). I do have multiple Pages around the size of yours though so it may have to be something I look into deeper if you're able to convert this into a nice revenue stream ![](data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7)

Also, we're adding the ability to schedule Reels to [FPTraffic](https://afflift.link/fptraffic). Let me know if you want to test it ![](data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7)

 [![](https://afflift.com/f/data/avatars/m/58/58177.jpg?1641647008)](https://afflift.com/f/members/tamas.58177/) 

*   [Add bookmark](https://afflift.com/f/posts/115890/bookmark "Add bookmark")
*   [#12](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115890)

**Update**

After DEADZ suggestions, I decided to go that route. Initially, I wanted to start a blog, add adsense or whatever but then I remembered that I did that a few times and just abandoned them...

This time, I want to be able to track as much as I can, not just blindly share the links like I always did... With the experience I've built since joining the forum (and my pop/push fail) I thought why not apply all the tips/tricks I've learned here... [trackers](https://afflift.com/f/link-directory/categories/tracking-tools.4/)/landers/scripts/funnels/everything.

Basically, my setup will be very similar to native ads but I'll add email to the mix too, using [Varun's tracking method](https://afflift.com/f/threads/how-to-track-where-your-email-subs-come-from-easily.8502/#post-115865).

This setup will also give me the advantage of moving my working funnels to paid ads easily in the future. Not sure yet, but that's my long-term _dream_ plan IF I'll find a working funnel.

So yesterday, I joined [skro](https://afflift.com/f/link-directory/skro-eu.206/)'s paid plan, bought a domain for my landing pages (then used [this](https://afflift.com/f/threads/host-your-landing-pages-for-free-w-cdn-in-10-minutes.7673/) and [this](https://afflift.com/f/threads/using-github-with-netlify-to-streamline-the-process.9145/) to set it up) and I've also signed up for a free [aweber](https://afflift.link/aweber) for now.

Today I've set up skro's clickbank integration and bought a custom domain.

I've then researched the [anstrex](https://afflift.com/f/link-directory/anstrex.24/) mega pack to find some nice landers I can adapt to my niche. A few blog style LP's a few short presell clickthru's and a few native style lp's so I can see what's working. 2 are already done.

Looks like it's working:

![](https://afflift.com/f/attachments/1680476137450-png.37225/)

That's it for now.

A lot of work but I feel like it'll be worth it more than pops. I'm actually building some real funnels now, not just some churn&burn pop spam.

I'm proud lol

In the coming days, I'll try to set up [Nick's guide too](https://afflift.com/f/threads/if-you%E2%80%99re-collecting-emails%E2%80%A6-you-need-to-do-this.10116/) as I'm sure I'll have a lot of landers and angles to test daily...

Any tips/tricks/questions?

 [![](https://afflift.com/f/data/avatars/m/58/58177.jpg?1641647008)](https://afflift.com/f/members/tamas.58177/) 

*   [Add bookmark](https://afflift.com/f/posts/115891/bookmark "Add bookmark")
*   [#13](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115891)

My in-stream ad earnings are starting to show up:

![](https://afflift.com/f/attachments/1680479479648-png.37226/)

I can't activate them on old videos. These stats are only from 12 videos I've uploaded since I've got accepted.

Also, it takes up to 7 days for earnings to show up :

![](https://afflift.com/f/attachments/1680479719206-png.37227/)

Interesting how this video took off only after 9 days:

![](https://afflift.com/f/attachments/1680479871074-png.37228/)

Another one is slowly dying after 3 days:

![](https://afflift.com/f/attachments/1680479965415-png.37229/)

but it made around $20 ![](data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7)

![](https://afflift.com/f/attachments/1680479993603-png.37230/)

![](https://afflift.com/f/attachments/1680480058003-png.37231/)

I think as the algo is testing ads on them AND if they can successfully drive conversions for advertisers they'll start pushing it to similar people. Wait, I'm sure, they always NEED more ad space... ![](data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7)

Has this one reached a T3 GEO? haha we'll see.

![](https://afflift.com/f/attachments/1680482553393-png.37234/)

Overall my page stats look awesome:

![](https://afflift.com/f/attachments/1680480513722-png.37232/)

Can't wait to pin some converting funnels to the top:

![](https://afflift.com/f/attachments/1680480625021-png.37233/)

Last edited: Apr 3, 2023

 [![](https://afflift.com/f/data/avatars/m/24/24970.jpg?1668026672)](https://afflift.com/f/members/noisyboy.24970/) 

*   [Add bookmark](https://afflift.com/f/posts/115900/bookmark "Add bookmark")
*   [#14](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115900)

Excited to see how this turns out. Good luck, [@Tamas](https://afflift.com/f/members/58177/).

Do you make the videos/reels yourself or just repurpose other people's content? How many posts/reels do you usually post per day at this point?

Your reach is looking awesome. Hope you're able to get some sales in! ![](data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7)

 [![](https://afflift.com/f/data/avatars/m/61/61235.jpg?1644765661)](https://afflift.com/f/members/andrew87.61235/) 

 [![](https://afflift.com/f/data/avatars/m/58/58177.jpg?1641647008)](https://afflift.com/f/members/tamas.58177/) 

*   [Add bookmark](https://afflift.com/f/posts/115912/bookmark "Add bookmark")
*   [#16](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115912)

> Excited to see how this turns out. Good luck, [@Tamas](https://afflift.com/f/members/58177/).
> 
> Do you make the videos/reels yourself or just repurpose other people's content? How many posts/reels do you usually post per day at this point?
> 
> Your reach is looking awesome. Hope you're able to get some sales in! ![](data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7)

Other people's content from my group and other platforms, using socialinsider for research and sometimes I also edit them using canva. If it's from a popular account I give credit in the description too.

I post 2-5 times a day in total a mix of videos,reels and memes.

 [![](https://afflift.com/f/data/avatars/m/0/1.jpg?1672673274)](https://afflift.com/f/members/luke.1/) 

*   [Add bookmark](https://afflift.com/f/posts/115955/bookmark "Add bookmark")
*   [#17](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115955)

I'm excited for you [@Tamas](https://afflift.com/f/members/58177/). I hope this is exactly what you've been looking for with results to more than make up for those other experiments ![](data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7)

 [![](https://afflift.com/f/data/avatars/m/77/77831.jpg?1681746680)](https://afflift.com/f/members/lazyguy.77831/) 

*   [Add bookmark](https://afflift.com/f/posts/115963/bookmark "Add bookmark")
*   [#18](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115963)

> The accounts are assets. Great that you kept them and kept at it. I'm excited to see how you monetize this and scale it.
> 
> Yes

Cool, I am gonna try to revive it.

> My in-stream ad earnings are starting to show up:
> 
> [View attachment 37226](https://afflift.com/f/attachments/37226/)
> 
> I can't activate them on old videos. These stats are only from 12 videos I've uploaded since I've got accepted.
> 
> Also, it takes up to 7 days for earnings to show up :
> 
> [View attachment 37227](https://afflift.com/f/attachments/37227/)
> 
> Interesting how this video took off only after 9 days:
> 
> [View attachment 37228](https://afflift.com/f/attachments/37228/)
> 
> Another one is slowly dying after 3 days:
> 
> [View attachment 37229](https://afflift.com/f/attachments/37229/)
> 
> but it made around $20 ![](data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7)
> 
> [View attachment 37230](https://afflift.com/f/attachments/37230/)  
> [View attachment 37231](https://afflift.com/f/attachments/37231/)
> 
> I think as the algo is testing ads on them AND if they can successfully drive conversions for advertisers they'll start pushing it to similar people. Wait, I'm sure, they always NEED more ad space... ![](data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7)
> 
> Has this one reached a T3 GEO? haha we'll see.
> 
> [View attachment 37234](https://afflift.com/f/attachments/37234/)
> 
> Overall my page stats look awesome:
> 
> [View attachment 37232](https://afflift.com/f/attachments/37232/)
> 
> Can't wait to pin some converting funnels to the top:
> 
> [View attachment 37233](https://afflift.com/f/attachments/37233/)

Wait does Facebook allows monetization on reused content?  
Or does it require some value addition in videos like YouTube ads under fair use policy?

 [![](https://afflift.com/f/data/avatars/m/58/58177.jpg?1641647008)](https://afflift.com/f/members/tamas.58177/) 

*   [Add bookmark](https://afflift.com/f/posts/115967/bookmark "Add bookmark")
*   [#19](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115967)

> Cool, I am gonna try to revive it.
> 
> Wait does Facebook allows monetization on reused content?  
> Or does it require some value addition in videos like YouTube ads under fair use policy?

It's a grey area... no rules like YT. Accounts are happy if we give them credit. Fb is happy for the adspace. I do edit most of them in canva.

 [![](https://afflift.com/f/data/avatars/m/58/58177.jpg?1641647008)](https://afflift.com/f/members/tamas.58177/) 

*   [Add bookmark](https://afflift.com/f/posts/115983/bookmark "Add bookmark")
*   [#20](https://afflift.com/f/threads/2k-month-facebook-page-monetization.10637/post-115983)

Video earnings looking good:

![](https://afflift.com/f/attachments/1680608462025-png.37264/)

I'll keep an eye on the stats to see what's working. So far 1 video is taking off.

![](https://afflift.com/f/attachments/1680608541852-png.37265/)

Campaigns:

![](https://afflift.com/f/attachments/1680608795852-png.37266/)

As you can see im tracking placements separately, I'll soon add reels comments meme comments and group posts into the mix too.

No [aweber](https://afflift.link/aweber) integration yet.

Also, forgot to mention the other day that I've setup my landers using [skro](https://afflift.com/f/link-directory/skro-eu.206/)'s no redirect option, it's working nicely. I thought if FB doesn't approve ad campaigns with redirects, why would they like it here?

![](https://afflift.com/f/attachments/1680610314531-png.37270/)

This is how the stats look on one of my campaigns. Pretty good reach AND engagement for a link post...

I'm testing 2 landers/offers now just to get a feel of the LP CTR and how the funnel works overall. I'll keep testing different content /offers until I find something that's working.

Need to work on my LP CTR.

![](https://afflift.com/f/attachments/1680609526780-png.37269/)

**Remember, this is all FREE, targeted T1 traffic.**

Now let's get that first InitiateCheckout.

Last edited: Apr 4, 2023
